tests/s3/test_object_workflow_operations.py

```diff
@@ -727,3 +768,233 @@ def test_delete_objects_and_mention_1000_objects_7657(self):
             cfg_7657["del_obj_cnt"])
         self.buckets_list.append(self.bucket_name)
         self.log.info("ENDED: Delete objects and mention 1000 objects.")
+
+    @pytest.mark.parallel
```

Comment:
Let's remove this marker across the new tests, since we have not performed any specific testing to validate the parallel test execution.

Reply:
Removed this marker in all the new tests